charts/intel/values.yaml

```diff
@@ -59,6 +59,10 @@ hqproperties:
   # default datacenter name is 'datacenter1'
   # hq.cassandra.db.localdatacenter: datacenter1
 
+# Optional property, allows setting java options for the hq container, e.g. "-Xmx1024m -Xms1024m"
+hq:
+  java_options: ""
```

Comment:
may be it should be commented and on the deployment.yaml have a if statment to check if it sets an empty value or it sets from the values.